.github/workflows/test-deploy.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - run: ./bin/dev config:set api_host https://api.dev.architect.io
       - run: ./bin/dev config:set log_level debug
       - run: ./bin/dev login -e ${{ secrets.ARCHITECT_EMAIL }} -p ${{ secrets.ARCHITECT_PASSWORD }}
-      - run: ./bin/dev register -a $ACCOUNT_NAME test/integration/hello-world/architect.yml test/mocks/superset/deprecated.architect.yml
```

Comment:
You shouldn't have to register the superset. That happens as part of the deploy.

Reply:
removed register step